### DISCUSSION TITLE
Neural Network Inquiry

### DISCUSSION THREAD

**Discussion by npiroozan** [mir-group/allegro#31] [Q&A] [answered]
Hello,

I hope you are doing well.  I was reviewing the Colab tutorial and the example for Silicon has intrigued me.  The AIMD dataset, upon which the neural network is trained, states to have been sampled for T=800K with 64 Si atoms.  The resultant plot after the LAMMPS simulation is an rdf distribution for T=300K.  How can this be if the input data does not cover 300K?

Apologies if I am misunderstanding something.

Thank you!

**Comment by simonbatzner** [comment, marked as answer]
Machine Learning Interatomic Potentials generalize well from high-temperature to low-temperature data. Data sampled at high-T cover a broad configuration space of diverse structures, from which it is often possible to generalize to lower-T data.

**Reply by npiroozan** [reply]
Got it, thank you.
